prototype2/multigrid/mgmesytec/Data.cpp

```diff
@@ -158,13 +163,24 @@ int MesytecData::parse(const char *buffer, int size, NMXHists &hists, ReadoutSer
     bytesleft -= 4;
```

Comment:
There is a misaligned pointer usage on the line that is `if ((*datap & 0x000000ff) != 0x58) {`.